src/components/simulation/SimulationCodeEditor.vue

```diff
@@ -1,15 +1,10 @@
 <template>
-  <div
-    class="simulationCodeEditor"
-    ref="simulationCodeEditor"
-    style="height: calc(100vh - 48px)"
-  >
-    <v-toolbar dense flat height="42">
+  <div class="simulationCodeEditor" ref="simulationCodeEditor" style="">
```

Comment:
Do we really need `style=""`? Or should it be removed?
```suggestion
  <div class="simulationCodeEditor" ref="simulationCodeEditor">
```